### PR TITLE
Install chromium from EPEL

### PR DESCRIPTION
**Pull request by dLobatog**
Offers an alternative to https://github.com/theforeman/foreman-infra/pull/392

### CODE REVIEW

File: puppet/modules/slave/manifests/init.pp [hunk 1]
```diff
@@ -232,23 +232,15 @@
...
-    yumrepo { 'google-chrome':
-      ensure   => present,
-      descr    => 'google-chrome - $basearch',
-      baseurl  => 'http://dl.google.com/linux/chrome/rpm/stable/$basearch',
-      enabled  => '1',
-      gpgcheck => '1',
-      gpgkey   => 'https://dl-ssl.google.com/linux/linux_signing_key.pub',
-      before   => Package['google-chrome-stable'],
+    package { 'chromium':
+      ensure => latest,
```
Comment (ekohl) [inline puppet/modules/slave/manifests/init.pp]: You might want to add a ```require => Class['epel']```

File: puppet/modules/slave/manifests/init.pp [hunk 2]
```diff
@@ -232,23 +232,15 @@
...
 
-    yumrepo { 'google-chrome':
-      ensure   => present,
-      descr    => 'google-chrome - $basearch',
-      baseurl  => 'http://dl.google.com/linux/chrome/rpm/stable/$basearch',
-      enabled  => '1',
-      gpgcheck => '1',
-      gpgkey   => 'https://dl-ssl.google.com/linux/linux_signing_key.pub',
-      before   => Package['google-chrome-stable'],
+    package { 'chromium':
```
Comment (ekohl) [inline puppet/modules/slave/manifests/init.pp]: You can simplify this and the next block to ```package { ['chromium', 'chromedriver']:```

**Comment by mmoll** [comment]
let's get this in now and tweak leter, if needed...

**Comment by mmoll** [comment]
merged, gracias @dLobatog!

**Pull request merged**